tests/3/expected.json

```diff
@@ -727,14 +726,6 @@
     }
   },
   "VXLAN_TUNNEL": {
-    "vtep": {
-      "src_ip": "10.0.0.2"
-    }
-  },
-  "VXLAN_TUNNEL_MAP": {
-    "vtep|map_104009_Vlan4009": {
-      "vlan": "Vlan4009",
-      "vni": "104009"
-    }
+    "vtep": {}
```

Comment:
I'm not sure if this is valid.

Reply:
Probably not.. should the VTEP be disabled when there is no loopback address?